home-screen-widget/profiles/JS_SDK.md

```diff
@@ -0,0 +1,15 @@
+### Javascript SDK API
+
+By calling these methods, you will receive a promise resolving with a the result of the operation.
+#### Messages
+``sdk.Messages.create(message)`` creates a new message 
+#### Profiles
+``sdk.Profiles.get(username)`` gets a profile by username
```

Comment:
we should advertise getting a profile by user ID as well, or instead.

Reply:
ok, this section I got from the README of the java script sdk repo. Maybe that was also never updated